src/logger/transports/index.mjs

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+import { console } from './console.mjs';
+import { github } from './github.mjs';
+
+export const transports = {
+  console,
+  github,
+};
+
+export const availableTransports = Object.keys(transports);
```

Comment:
```suggestion
export {
  console,
  github,
};
```

Can't we just get the Object.keys in the consumer?

Reply:
It's used more than once, so I prefer to maintain the DRY principle